daemon/src/db/cull_dlcs.rs

```diff
@@ -0,0 +1,187 @@
+use crate::db::Connection;
+use anyhow::Result;
+use model::EventKind;
+
+impl Connection {
+    #[allow(dead_code)]
+    async fn cull_old_dlcs(&self) -> Result<()> {
+        let mut conn = self.inner.acquire().await?;
+
+        let rows = sqlx::query!(
+            r#"
+SELECT
+    id,
+    name,
+    data
+FROM
+    events
+WHERE
+    name = $1 OR
+    name = $2
+ORDER BY
+    created_at DESC
+LIMIT -1 OFFSET 1
```

Comment:
This should not be necessary, i.e. you want to select all anyway and `limit -1, offset 1` would _limit_ your results from index `-1`, i.e. all :)

Reply:
I will try without `LIMIT -1`, but I read that it was needed for SQLite.

Reply:
I get this error as soon as I remove the `LIMIT -1`:

```
error: error returned from database: near "OFFSET": syntax error
  --> daemon/src/db/cull_dlcs.rs:10:20
   |
10 |           let rows = sqlx::query!(
   |  ____________________^
11 | |             r#"
12 | | SELECT
13 | |     id,
...  |
26 | |             model::EventKind::ROLLOVER_COMPLETED_EVENT,
27 | |         )
   | |_________^
   |
```

Source for the `LIMIT -1` solution: https://stackoverflow.com/a/19676495.

Reply:
You'd need to remove the `offset` as well which should give you the whole table.

Reply:
But the point was to omit the newest rollover/contract-setup event, because we don't want to cull that one.

I can do that outside of course. Originally it mattered more to do it in the query, because I was attempting to modify the `data` field inside of the query. It could not work because the data field is a JSON blob so it seems impossible to tweak a particular field in there.

Reply:
Oh, so sorry then. I did not understand your intention.